src/components/Activity/NewRequestsRow.js

```diff
@@ -95,9 +98,12 @@ class NewRequestsRow extends Component {
               <View style={styles.horizontalSpaceBetween}>
                 <View style={styles.userNameWithTime}>
                   <Text style={styles.name}>{this.props.dataSource.name}</Text>
-                  <Text style={styles.fromNow}>{moment(this.props.dataSource.request_date).fromNow()}</Text>
+                  <Text style={styles.fromNow}>
+                    {moment(this.props.dataSource.request_date).fromNow()}</Text>
```

Comment:
'{moment(this.props.dataSource.request_date).fromNow()}</Text>'
might be changed into
'  {moment(this.props.dataSource.request_date).fromNow()}
</Text>'